core_lib/src/util/util.h

```diff
@@ -51,4 +51,6 @@ Container filter(const Container& container, Pred predicate) {
 QString ffprobeLocation();
 QString ffmpegLocation();
 
+quint64 imageSize(const QImage&);
```

Comment:
This method should be const, it's not something we apply often but it does make it easier to understand that the content of this shouldn't be modified.

Reply:
This is a simple function, not a class member. I have put a const for the QImage

Reply:
Fair enough, I like to do it whenever possible for readability sake but it's not a problem to me if we don't do it here.